packages/webhooks/schema/models/WebhookPayload.js

```diff
@@ -0,0 +1,168 @@
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const {
+    WEBHOOK_PAYLOAD_STATUSES,
+    WEBHOOK_PAYLOAD_STATUS_PENDING,
+    WEBHOOK_PAYLOAD_TTL_DAYS,
+    WEBHOOK_PAYLOAD_MAX_RESPONSE_LENGTH,
+} = require('@open-condo/webhooks/constants')
+const access = require('@open-condo/webhooks/schema/access/WebhookPayload')
+const { encryptionManager } = require('@open-condo/webhooks/utils/encryption')
+const { validateEventType } = require('@open-condo/webhooks/utils/validation')
+
+const ERRORS = {
+    INVALID_EVENT_TYPE: {
+        code: BAD_USER_INPUT,
+        type: 'INVALID_EVENT_TYPE',
+        message: 'Invalid event type format',
+    },
+}
+
+const WebhookPayload = new GQLListSchema('WebhookPayload', {
+    schemaDoc: 'Stores webhook payloads for sending. Contains ready-to-send payload, URL, and secret. Tracks sending status, retries, and responses from external servers.',
+    fields: {
+
+        url: {
+            schemaDoc: 'Where to send the webhook',
+            type: 'Url',
+            isRequired: true,
+        },
+
+        payload: {
+            schemaDoc: 'JSON data to send. Stored encrypted in database.',
+            type: 'EncryptedText',
+            encryptionManager,
+            isRequired: true,
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+        },
+
+        secret: {
+            schemaDoc: 'Secret key for signing requests (HMAC). Stored encrypted in database.',
+            type: 'EncryptedText',
+            encryptionManager,
+            isRequired: true,
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+        },
+
+        eventType: {
+            schemaDoc: 'Type of event that triggered this webhook. Use dot-notation format: "{resource}.{action}" or "{resource}.{sub-resource}.{action}". Examples: "payment.created", "payment.status.changed", "invoice.paid", "user.deleted". Use lowercase with dots as separators.',
+            type: 'Text',
+            isRequired: true,
+            hooks: {
+                validateInput: ({ resolvedData, fieldPath, context }) => {
+                    const value = resolvedData[fieldPath]
+                    if (value) {
+                        const { isValid, error } = validateEventType(value)
+                        if (!isValid) {
+                            throw new GQLError({ ...ERRORS.INVALID_EVENT_TYPE, message: error }, context)
+                        }
+                    }
+                },
+            },
+        },
+
+        modelName: {
+            schemaDoc: 'Name of the model that triggered this webhook (e.g., "Payment")',
+            type: 'Text',
+            isRequired: false,
+        },
+
+        itemId: {
+            schemaDoc: 'ID of the record that triggered this webhook',
+            type: 'Uuid',
+            isRequired: false,
+        },
```

Comment:
Should we add optional `subscription` field, which can help us debug if something is missing